Readme.md

```diff
@@ -162,13 +162,17 @@ options.
 
 ## API
 
-### Airbreak.createClient(apiKey, [env])
+### var airbrake = Airbrake.createClient(key, [env])
```

Comment:
Docs are important! Thanks for your help making them better @jadengore :+1: :sparkles: